src/PointwiseFunctions/AnalyticSolutions/RelativisticEuler/RotatingStar.cpp

```diff
@@ -600,7 +677,9 @@ RotatingStar::variables(
     const tnsr::I<DataType, 3>& /*x*/,
     tmpl::list<hydro::Tags::RestMassDensity<DataType>> /*meta*/) const {
   interpolate_vars_if_necessary(vars);
-  return {Scalar<DataType>{vars->rest_mass_density.value()}};
+  using std::max;
+  return {Scalar<DataType>{
+      DataType{max(atmosphere_floor_, vars->rest_mass_density.value())}}};
```

Comment:
@nilsdeppe The problem has been fixed here. `RotatingStar` only uses 1D EoS to initialize currently, so any FPE issues can be taken care of by setting the floor on the density when it's first loaded in. Thus, the floor is only applied once.
(We discussed in person but I'm noting it here for the good of the cause)